addons/sourcemod/scripting/surftimer/misc.sp

```diff
@@ -4408,6 +4408,48 @@ public void TeleportToSaveloc(int client, int id)
 	DispatchKeyValue(client, "targetname", g_szSaveLocTargetname[id]);
 	SetEntPropVector(client, Prop_Data, "m_vecVelocity", view_as<float>( { 0.0, 0.0, 0.0 } ));
 	TeleportEntity(client, g_fSaveLocCoords[id], g_fSaveLocAngle[id], g_fSaveLocVel[id]);
+	CreateTimer(0.1, noPrac, client);
```

Comment:
Please use convert the client index to the userid or serial index with `GetClientUserId` or `GetClientSerial` to prevent race conditions.  To get back the client index use `GetClientFromSerial` or `GetClientOfUserId` and check if the client index is still valid.